File: tarefa1-JavaScript/controllers/loginController.js
const User = require('../models/user');
const login = (req, res) => {
const { username, password } = req.body;
const user = new User(username, password);
if (user.validate()) {
res.redirect('/welcome');
} else {
res.redirect('/error');
}
};
module.exports = {
login,
};

File: mvc_project/controllers/login_controller.py
# controllers/login_controller.py
from flask import redirect, request, url_for
# Função para validar o login
def login():
 username = request.form['username']
 password = request.form['password']
# Simula a autenticação, substitua pela lógica real
 if username == 'Admin' and password == '123456':
    return redirect(url_for('welcome'))
 else:
    return redirect(url_for('error'))